tests/unit/cli/cli-test.js

```diff
@@ -66,11 +79,21 @@ function stubRun(name) {
 
 describe('Unit: CLI', function() {
   beforeEach(function() {
-    willInterruptProcess = td.replace('../../../lib/utilities/will-interrupt-process', {
-      addHandler: td.function(),
-      removeHandler: td.function(),
+
+    _process = new MockProcess({
+      exit() {
+        // force `process-exit` to handle exit
+        process.exit();
+      },
     });
 
+    // capture-exit doesn't support `process` injection
+    // instead it always uses global `process`
+    origExit = process.exit;
+    // if we leave original exit then test run will be stopped once we send SIGINT
+    process.exit = function() {};
```

Comment:
I've tried to explain the intention for this juggling with `exit` in comments. Basically the problem is that currently we can't path `process` instance to the `capture-exit.captureExit()`.

Btw I prefer to avoid changes in the `capture-exit` in the scope of this pr.